src/main/java/org/zeromq/ZContext.java

```diff
@@ -31,6 +35,11 @@
     private List<Socket> sockets;
 
     /**
+     * List of selectors managed by this ZContext
+     */
+    private List<Selector> selectors;
+
+    /**
```

Comment:
It would be possible to handle the list of selectors inside the Ctx class, reducing the amount of duplicated code and the 3 layers of context (Ctx, Context, ZContext) would benefit of the mechanism.